js/map.js

```diff
@@ -52,15 +52,27 @@ function randomPassableTile(){
 
 function generateMonsters(){
     monsters = [];
-    let numMonsters = level+1;
+    const numMonsters = level > 5 ? level+1 : 6;
     for(let i=0;i<numMonsters;i++){
         spawnMonster();
     }
 }
 
 function spawnMonster(){
     // let monsterType = shuffle([Goose, Ant, Mushroom, Eater, Toast])[0];
-    let monsterType = shuffle([Rock, Paper, Scissors ,Rock_Plus, Paper_Plus, Scissors_Plus, Rock_Anti, Paper_Anti, Scissors_Anti])[0];
-    let monster = new monsterType(randomPassableTile());
-    monsters.push(monster);
+  const monsterTypes = [Rock, Paper, Scissors, Rock_Plus, Paper_Plus, Scissors_Plus, Rock_Anti, Paper_Anti, Scissors_Anti];
+  const monsterType = randomMonsterType( monsterTypes );
+  const monster = new monsterType(randomPassableTile());
+  monsters.push(monster);
 }
+
+function spawnWeakMonster(){  
+  const monsterTypes = [Rock, Paper, Scissors];
+  const monsterType = randomMonsterType( monsterTypes );
+  const monster = new monsterType(randomPassableTile());
+  monsters.push(monster);
+}
+
+function randomMonsterType(monsterTypes) {
+  return shuffle(monsterTypes)[0]
+}
```

Comment:
You should add a new line at the end of file! you can configure your editor to do it for you and never worry about it

Reply:
good call